src/uct/tcp/tcp_sockcm_ep.c

```diff
@@ -919,20 +919,15 @@ static ucs_status_t uct_tcp_sockcm_ep_server_create(uct_tcp_sockcm_ep_t *tcp_ep,
 
     UCS_ASYNC_BLOCK(async);
 
-    UCS_CLASS_CLEANUP(uct_cm_base_ep_t, &tcp_ep->super);
-
-    /* set the server's ep to use the cm from params and its iface
+    /* set the server's ep to use the cm from params.
      * (it could be the previous one it had - the one used by the listener or
      * a new one set by the user) */
-    status = UCS_CLASS_INIT(uct_cm_base_ep_t, &tcp_ep->super, params);
+    status = uct_cm_set_common_data(&tcp_ep->super, params);
```

Comment:
pls rename `uct_cm_set_common_data` to `uct_cm_ep_base_set_common_data`

Reply:
@evgeny-leksikov this function is currently defined in uct_cm.h, same as uct_cm_ep_pack_cb and uct_cm_ep_resolve_cb (which dont have _base_ in their name).
if i rename to uct_cm_ep_base_set_common_data then it would make sense to move it to uct_iface.h but imo it's more suited in uct_cm.h. maybe just uct_cm_ep_set_common_data like the other functions there?
wdyt?